app/src/main/res/navigation/nav_demo.xml

```diff
@@ -36,7 +38,8 @@
             app:argType="integer" />
         <action
             android:id="@+id/toMoreDetailsFragment"
-            app:destination="@id/moreDetailsFragment" />
+            app:destination="@id/moreDetailsFragment"
+            app:enterAnim="@anim/scale_animation_fragment" />
```

Comment:
just a reminder that there are exitAnim, popEnterAnim, popExitAnim, you can use them to animate the back button pressed